packages/plugins/rows/src/index.tsx

```diff
@@ -22,6 +22,8 @@ export function createRowsPlugin(
 ): EditorPlugin<RowsPluginState, RowsConfig> {
   const { content } = config
 
+  console.log('Banana') // @@@
```

Comment:
🍌 😂

Reply:
love it, way better than the test strings I usually use.

Reply:
Damn! Just tried to remove it but you were too quick! :laughing: 
Banana is just a nice word to type.